app/styles/partials/footer.scss

```diff
@@ -33,3 +33,68 @@ footer {
     right: 0 !important;
   }
 }
+
+
+@media screen and (min-width : 1200px) {
+    #main-container {
```

Comment:
@niranjan94 I have overrided the classes of semantic ui. So any rails with those classes will be aligned as is present in above pictures. Adding placeholders will improve the quality of code. That's it. I am working on it. Still, I guess without placeholders the code will also work.